cmd/allocator/main.go

```diff
@@ -107,13 +113,24 @@ func main() {
 		for {
 			select {
 			// watch for events
+			case event := <-watcherTLS.Events:
+				h.tlsMutex.Lock()
+				tlsCert, err := readTLSCert()
+				if err != nil {
+					logger.WithError(err).Error("could not load TLS cert; keeping old one")
+				} else {
+					h.tlsCert = tlsCert
```

Comment:
This is the only line that needs the mutex guard, right?

Reply:
Yup, fixed this in (https://github.com/googleforgames/agones/pull/1638).